internals/daemon/daemon_test.go

```diff
@@ -98,6 +98,29 @@ func (h *fakeHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	h.lastMethod = r.Method
 }
 
+func (s *daemonSuite) TestRegisterCommand(c *C) {
```

Comment:
As discussed, register -> add change in general as we are now exposing Api list.